### PR TITLE
Changes cloaks into exosuit clothes.

### PR DESCRIPTION
**Pull request by TechnoAlchemisto**
I think I asked Xhuis to do this at some point so I decided to try and do it myself.

### CODE REVIEW

File: code/modules/clothing/suits/cloaks.dm [hunk 1]
```diff
@@ -14,39 +14,39 @@
...
 	icon_state = "hoscloak"
-	armor = list(melee = 10, bullet = 10, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0)
+	armor = list(melee = 20, bullet = 15, laser = 15, energy = 10, bomb = 25, bio = 0, rad = 0)
 
-/obj/item/clothing/cloak/qm
+/obj/item/clothing/suit/cloak/qm
 	name = "quartermaster's cloak"
 	desc = "Worn by Cargonia, supplying the station with the necessary tools for survival."
-
+	armor = list(melee = 0, bullet = 15, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0)
```
Comment (francinum) [inline code/modules/clothing/suits/cloaks.dm]: HIDDEN CHANGES

Comment (TechnoAlchemisto) [inline code/modules/clothing/suits/cloaks.dm]: Cargonia forever

**Comment by lordpidey** [comment]
Boo!  Now how will I wear my CMO jacket and cloak?

**Comment by Iamgoofball** [comment]
:-1: why not make it an attachment instead, no hos will wear his cape over this

**Comment by TechnoAlchemisto** [comment]
Armor values were buffed on HoS cloak to make it not a throw away. Still, fashion has it's price.

**Comment by Iamgoofball** [comment]
Quite frankly, that's a retarded reason, and you should fucking see why.

Why should I be penalized for wanting to take a cosmetic option?

File: code/modules/clothing/suits/cloaks.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/modules/clothing/suits/cloaks.dm]: This change doesn't even make any sense. Why would the QM's cloak have bullet resist on it? He's nowhere near bullets 99% of the time. Melee? At least that somewhat makes sense, due to mining, but...really...why does the QM need armor, at all?

**Comment by TechnoAlchemisto** [comment]
Was not having a backpack/satchel not a penalty?

File: code/modules/clothing/suits/cloaks.dm (hunk 1 above)
Comment (TechnoAlchemisto) [inline code/modules/clothing/suits/cloaks.dm]: Cheeky commentary on gun cargo.

**Comment by Dorsisdwarf** [comment]
Hey remember when the cosmetic options were weaker than the normal items across the board so you only took them to show off how robust you were/ that you didnt care about winning?

**Comment by Iamgoofball** [comment]
@TechnoAlchemisto 
Yes, and the point of this PR was to fix that, but you fucked up and made it just as bad.

@Dorsisdwarf 
cool beans, but maybe if we didn't have this retarded attitude of "if you want to have a decent loadout you have to look like shit, FASHION HAS A COST YOU KNOW" we could JUST HAVE BOTH

**Comment by Dorsisdwarf** [comment]
We dont have that anymore. Winter coats are stealth armor vests, and sec caps and berets were until like, last week, stealth helmets.
The HOS' eyepatchHUD used to have no flash protect (Because it didnt cover your fucking eyes) but that was changed too

**Comment by Iamgoofball** [comment]
As an HoS, why would I EVER use the winter coat over anything else? Literally anything else?

Why is it such a huge fucking problem for us to just support **having both at the same time**

**Comment by Dorsisdwarf** [comment]
That was the whole point of my snark, pointing out that all this shit used to be an optional weaker thing you took to show off, instead of a purely cosmetic difference

**Comment by Iamgoofball** [comment]
oh

sometimes you can't tell anymore if some things are serious or not in this community

**Comment by TechnoAlchemisto** [comment]
You do have a valid point goof, the intent of this PR was to make cloaks more desirable. I mean after all, I did design them and would like them to be used more.

It seems the main issue here is the HoS cloak, since the great coat covers the same limbs and the cloak is essentially just less armored. 

The fixes, if this truly is such a bad PR, would be to let the cloak remain a back item, but give it 2-3 storage spaces(since cloaks often have pockets inside) or to make it an exosuit with identical armor values to corresponding items. In which case it would make the HoS have two additional greatcoats to give away.

**Comment by TechnoAlchemisto** [comment]
(Although I always liked that fancy items were less robust, thematically. It's always fun to be a show off if you are that comfortable)

**Comment by Iamgoofball** [comment]
Honestly, I think the BEST fix here would be to literally just make cloaks Attachment items like Ties and the Armbands.

**Comment by TechnoAlchemisto** [comment]
I mean that would make them more like a cape to be fair. Cloaks wrap around the body and are worn as an outer layer.  

I wouldn't be against them being an attachment but I 1: have no idea how to do that and 2. Think cloaks are a bit much as attachments and detract from the overall intention of the cloaks.

File: code/modules/clothing/suits/cloaks.dm [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
-/obj/item/clothing/cloak
+/obj/item/clothing/suit/cloak
 	name = "brown cloak"
 	desc = "It's a cape that can be worn on your back."
 	icon = 'icons/obj/clothing/cloaks.dmi'
 	icon_state = "qmcloak"
 	w_class = 2
-	slot_flags = SLOT_BACK
+	slot_flags = SLOT_OCLOTHING
```
Comment (Jordie0608) [inline code/modules/clothing/suits/cloaks.dm]: typo

**Pull request closed without merging**